guide/daily-spending-wallet/backup-and-recovery/manual-backup.md

```diff
@@ -21,64 +21,39 @@ Illustration sources:
 -->
 
 {% include picture.html
-image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/backing-up-a-wallet-manual.png"
-retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/backing-up-a-wallet-manual@2x.png"
-mobile = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/backing-up-a-wallet-manual-mobile.png"
-mobileRetina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/backing-up-a-wallet-manual-mobile@2x.png"
-alt-text = "Backing up a recovery phrase chapter header image"
-width = 1600
-height = 600
-layout = "full-width"
+   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/backing-up-a-wallet-manual.png"
+   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/backing-up-a-wallet-manual@2x.png"
```

Comment:
Should link to this the [deleting the recovery phrase](https://deploy-preview-892--bitcoin-design-site.netlify.app/guide/daily-spending-wallet/security/#deleting-the-recovery-phrase) section in the security page somewhere on this page

Reply:
Where within the text does it make the most sense?

Reply:
Probably at the very end as it's the last step you'd take (if you take it at all). Maybe even segment it into an Advanced section like me and Christoph have done with sending / requesting page